pipelines/tekton/aiedge-e2e/aiedge-e2.model-registry.tensorflow-housing.pipelinerun.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  labels:
+    tekton.dev/pipeline: aiedge-e2e
+  generateName: aiedge-e2e-
+spec:
+  params:
+  - name: model-name
+    value: tensorflow-housing
+  - name: model-version
+    value: "2"
+  - name: s3-bucket-name
+    value: rhoai-edge-models
+  - name: model-registry-hostname
+    value: "modelregistry-sample-http-default.apps.srbiswas-cluster2.dev.datahub.redhat.com"
+  - name: git-containerfile-repo
+    value: https://github.com/opendatahub-io/ai-edge.git
+  - name: model-registry-enabled
+    value: "true"
```

Comment:
Just an idea for better readability, try to move this model-registry param to the model-registry-hostname, so it is grouped.